fem/datacollection.hpp

```diff
@@ -410,9 +410,14 @@ class VisItFieldInfo
    std::string association;
    int num_components;
    int lod;
+   std::string basis;
+   int order;
    VisItFieldInfo() { association = ""; num_components = 0; lod = 1;}
-   VisItFieldInfo(std::string association_, int num_components_, int lod_ = 1)
-   { association = association_; num_components = num_components_; lod =lod_;}
+   // CYRUS Question: Would it make sense to reorder?
```

Comment:
What is the proper order of args?

Reply:
If the question is about preferred order of the parameters, I think it does not matter much as the class is used only internally and the constructor is always called with all arguments (after the recent changes). So I would vote for keeping it as it is and removing the comment.